app/views/layouts/devise.html.slim

```diff
@@ -0,0 +1,20 @@
+doctype html
+html.layout-devise lang=I18n.locale
+  head
+    title
+      = page_title
+    == csrf_meta_tags
+    == csp_meta_tag
+
+    == javascript_pack_tag 'application'
+    == stylesheet_pack_tag 'application'
+
+  body.h-screen.bg-gray-900.bg-image
+    main.h-full.w-full.grid.place-center.main-overlay
+      .w-full.max-w-xs.md:max-w-xs
+        .h-10 = svg_tag(icon_id: 'icon-logo-white', html: { class: 'w-full h-full' })
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/b2f39155e54d78bea577b04a53a9496122f58f84/app/views/layouts/devise.html.slim#L15"></span>Line is too long. [85/80]</td>
    </tr>
  </tbody>
</table>